server/config/config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
     development: {
         host: "localhost",
         port: 5000,
-        url: "http://" + this.host + "/" + this.port,
+        url: "https://" + this.host + "/" + this.port,
```

Comment:
Should this still stay? I thought we had established that developers don’t need to switch to https anymore?

Reply:
yes, it does not matter, I can change it back